_posts/2018-05-24-mentorship-open-source-hackillinois.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
```

Comment:
Would it be possible to start off with a brief 1-2 sentence intro to HackIllinois and its focus on open source. Such a prelude would help to orient readers who may not be familiar with HackIllinois (e.g., new sponsors, students, mentors).

> Open Source

How do we intend to stylize "Open Source"? Capitalization or not works fine for me, but we should be consistent. I was thinking of starting a style guide so we could consolidate our terminology and provide a jumping off point for new team members.

> 25 sponsors

Similar to what was done for mentors, let's link to the sponsors.

> has transitioned

As a nit, if we consider the transition to be complete, then use the simple past tense "transitioned".

> community; to encourage

Use a colon (:) as the separator as the phrase following is not a sentence in and of itself.